.gitlab-ci.yml

```diff
@@ -220,9 +220,9 @@ Basic Readout:
   script:
     - source ${CI_PROJECT_DIR}/install/thisREST.sh
     - cd ${CI_PROJECT_DIR}/projects/basic-readouts/
+    - git submodule update --init
+    - git pull
     - ls
-    - git status
-    - git log
     - restRoot --m 1 -b -q GenerateReadouts.C'("basic.root")'
```

Comment:
Can you also remove the `--m 1`, as far as I understand this is a custom macro

Reply:
True, it is not needed here. What about the online edition and aprouval of minor changes? I believe @lobis commented this out somewhere